db/seeds.rb

```diff
@@ -11,10 +11,10 @@
 require "#{Rails.root}/db/seeds/database_constants"
 
 puts "Creating demo schools, homerooms, interventions..."
-raise "empty yer db" if School.count > 0 ||
-                        Student.count > 0 ||
-                        InterventionType.count > 0 ||
-                        Assessment.count > 0
+# raise "empty yer db" if School.count > 0 ||
```

Comment:
Can we remove these comments?  This seems like a good guardrail in case someone actually runs this on a non-empty database.  If a migration https://github.com/studentinsights/studentinsights/pull/1178 puts records in the db seems better to tackle that directly (and this helped catch that!).

Reply:
OK! I realized after i made the PR that I could change `Assessment.count > 2` would that be ok?

Reply:
Ah I misread this earlier, sorry!

I think that there are three issues:
- we do want the seed task to check for data in the database and abort
- the check here is out of data and has been for a while but we didn't notice
- the migration @alexsoble added revealed this accidentally

For this pull request, why don't we just change these lines to:
```
raise "empty yer db" if School.count > 0 || Student.count > 0 || EventNote.count > 0 || Service.count > 0
```

so we can ship your awesome improvements to reduce the total record count.